wavefront/server/packages/flo_cloud/flo_cloud/azure/synapse.py

```
@@ -0,0 +1,252 @@
+        converted_query = re.sub(r'@(\w+)', replace_param, query)
+        return converted_query, ordered_values
+
+    def execute_query_as_dict(
+        self, query: str, params: Optional[Dict[str, Any]] = None
+    ) -> List[Dict[str, Any]]:
+        converted_query, values = self._convert_named_params(query, params)
+        with self.get_cursor() as cursor:
+            try:
+                cursor.execute(converted_query, values)
+                columns = [desc[0] for desc in cursor.description]
+                return [dict(zip(columns, row)) for row in cursor.fetchall()]
```

Ruff (0.15.7)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)